Marlin/Marlin_main.cpp

```diff
@@ -494,11 +494,6 @@ static bool send_ok[BUFSIZE];
 
 #if HAS_SERVOS
   Servo servo[NUM_SERVOS];
-  #define MOVE_SERVO(I, P) servo[I].move(P)
-  #if HAS_Z_SERVO_ENDSTOP
-    #define DEPLOY_Z_SERVO() MOVE_SERVO(Z_ENDSTOP_SERVO_NR, z_servo_angle[0])
-    #define STOW_Z_SERVO() MOVE_SERVO(Z_ENDSTOP_SERVO_NR, z_servo_angle[1])
-  #endif
 #endif
```

Comment:
Rarely used. And no significant improvement.

Needed the literal `servo[Z_ENDSTOP_SERVO_NR].move(z_servo_angle[((stow) ? 1 : 0)])` to use a ternary.